test/framework/setup.go

```diff
@@ -214,6 +214,17 @@ func NamespaceWithSidecarInjection(namespace string) InstallFunc {
 	return YamlK8s(fmt.Sprintf(`
 apiVersion: v1
 kind: Namespace
+metadata:
+  name: %s
+  labels:
+    kuma.io/sidecar-injection: "enabled"
+`, namespace))
+}
+
+func NamespaceWithSidecarInjectionOnAnnotation(namespace string) InstallFunc {
```

Comment:
Maybe add a comment to explain why this is there (probably even put a deprecated comment there to avoid accidental use of it).